### PR TITLE
[CCTRI-1885] Add Patch functionality

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**

### CODE REVIEW

File: src/container/Content/components/ModalForPatch/ModalForPatch.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import { connect } from "react-redux";
+
+
+const ModalForPatch = (props) => (
+    <div className='modal' id='modal'>
+        <Formik
+            initialValues={{ client_id: '', password: '', module_type_id: '', iroh_service_url: '' }}
+            validate={values => {
+                const errors = {};
+                if (!values.client_id) {
```
Comment (mr-pavlenko) [inline src/container/Content/components/ModalForPatch/ModalForPatch.js:17]: if you need a new array of errors you can use the map. Why do you need it?

File: src/sagas.js [hunk 2]
```diff
@@ -78,5 +83,28 @@ function* pushSagaWorker(action) {
...
+function* patchSagaWorker (action) {
+    try {
+        yield put(showLoader())
+        const token = yield call(authorize, action.payload)
+        const url = action.payload.iroh_service_url
+        const json = yield call(pullModuleType, action.payload, token)
+        yield put(onPatchModuleTypeSuccess(json, url))
+    }
+    catch (e){
+        yield put(onPullModuleTypeError(String(e).replace(/^\w/, c => c.toUpperCase())))
```
Comment (mr-pavlenko) [inline src/sagas.js:95]: Why do you need to replace there?

Comment (mr-pavlenko) [reply src/sagas.js:95]: you can take e.message

Comment (AnastasiiaRozlyvan) [reply src/sagas.js:95]: @mr-pavlenko `e.message` is empty in this case

**Pull request merged**